Formula/emacs.rb

```diff
@@ -94,10 +81,15 @@ def install
 
     if build.with? "cocoa"
       args << "--with-ns" << "--disable-ns-self-contained"
-      system "./configure", *args
-      system "make"
-      system "make", "install"
+    else
+      args << "--without-x" << "--without-ns"
+    end
```

Comment:
Just curious, why `--without-x` only when building without Cocoa support? I know that's the original logic, but I wonder if it's appropriate.